Makefile

```diff
@@ -1,13 +1,18 @@
 SHELL := /bin/bash
+POSTGRES_PASSWORD := postgrespassword
+PORT := 5432
+ADMIN_PORT := 80
+PGADMIN_DEFAULT_EMAIL := postgres@gmail.com
```

Comment:
Kind of strange, but I know you're just moving the address that was there before. Normally I'd use something in our own domain or example.com